Tesla_rm/Tesla_rm/View/StartView.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  StartView.swift
+//  Tesla_rm
+//
+//  Created by Дмитрий Супрун on 31.01.23.
+//
+
+import SwiftUI
+
+/// Start (lock/unlock) Tesla
+struct StartView: View {
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let lockTeslaImageName = "Lock Tesla"
+        static let unlockTeslaImageName = "Unlock Tesla"
+        static let backgroundColorName = "Background"
+        static let settingsImageName = "Gear"
+        static let closeTextName = "Close"
+        static let openTextName = "Open"
+        static let unlockImageName = "lock.open.fill"
+        static let lockImageName = "lock.fill"
+        static let hiTextName = "Hi"
+        static let welcomeTextName = "Welcome Back"
+    }
+    
+    // MARK: - Public Properties
+    
+    var body: some View {
+        NavigationView {
+        backgroundZStackView {
+                VStack {
+                    ZStack {
+                        ZStack {
+                            radialLightGradient
+                            Image(Constants.unlockTeslaImageName)
+                                .resizable()
+                                .scaledToFit()
+                                .shadow(radius: 300)
+                            VStack {
+                                Text(Constants.hiTextName)
+                                    .font(.title)
+                                Text(Constants.welcomeTextName)
+                                    .font(.largeTitle)
+                                    .fontWeight(.bold)
+                            }
+                            .offset(y: -200)
+                        }
+                        .opacity(startViewModel.isUnlocked ? 1 : 0)
+                        ZStack {
+                            radialBlackGradient
+                            Image(Constants.lockTeslaImageName)
+                                .resizable()
+                                .scaledToFit()
+                                .shadow(radius: 300)
+                            
+                        }
+                        .opacity(startViewModel.isUnlocked ? 0 : 1)
+                        
+                        settingButtonView
+                            .offset(y: -320)
+                        
+                        unlockCarView
+                            .offset(y: 200)
```

Comment:
```suggestion
                        settingButtonView
                        
                        unlockCarView
                          
```
офсеты убери внутрь тела свойства